### ISSUE TITLE
Include an 'all_objects' manager

### ISSUE THREAD

**Issue by diegooquendoopalytica** [san4ezy/django_softdelete#8]
For my specific problem was necessary to show all objects in django admin tables, including the deleted ones, because by default the admin.py gets the `Model.objects` and that manager is already filtered.

I was able to achieve that using this simple code

```python
# In my model.py

class AllObjectsManager(models.Manager):
    def get_queryset(self):
        return models.query.QuerySet(self.model, self._db).all()

class Employee(SoftDeleteModel):
    # Manager to list all objects including deleted ones
    all_objects = AllObjectsManager()
    # Attributes bellow

# In my admin.py

@admin.register(Employee)
class EmployeeAdmin(admin.ModelAdmin):

    def queryset(self, request):
        qs = super(Employee, self).get_queryset(request)
        return qs.all_objects()
```

Would be nice to include this by default :D

**Comment by a-edakin** [comment]
Hi @diegooquendoopalytica !


I can't provide you with link to this repo because it's outdated #9 , but in last release 0.9.21 there is `global_manager`

![image](https://user-images.githubusercontent.com/22201283/194069241-6102c791-a99c-474a-8f39-b1c578e9eafc.png)


So you can just use it.

**Comment by san4ezy** [comment]
Hi, guys! Sorry, I wasn`t be able to reply a long time. 

Currently the repo is updated to the last version. And yes, @diegooquendoopalytica good idea, but this functionality is available since version 0.9.21 (the last version on PyPI). It does not documented yet, but you can use it as @a-edakin `s commented.

I will add documentation soon, or you could do it, if you want :)

Thanks!
